src/Xam.Plugin.LatestVersion.csproj

```diff
@@ -12,7 +12,7 @@
     <!-- NuGet package info -->
     <PackageId>Xam.Plugin.LatestVersion</PackageId>
     <Title>LatestVersion Plugin for Xamarin and Windows</Title>
-    <PackageVersion>0.1.0</PackageVersion>
+    <PackageVersion>0.1.1</PackageVersion>
```

Comment:
Please roll this change back as well. The package version is set by the build pipeline. Thanks!

Reply:
Ok.
Yes, the sln changes are not required... but it's automatically modified by my VS2022... I'll try to rollback by modifying the sln manually...